Templates/TodoList-NodeJS/readme.md

```diff
@@ -0,0 +1,14 @@
+## Todo List with Node.js via HttpTrigger & Storage Table
+HttpTrigger allows you to implement CRUD implementation over Rest API. This template shows how to use a httptrigger create and read entities that resembles todo list entities from Storage Table.
+
+## How it works
+The http methods are mapped to CRUD operation on an Azure Storage Table binding. `GET` in this function is mapped to `Read all entities` and `POST` is mapped to `Create a new entity`. Http POST expects JSON data as shown below.
+```JSON
+{
+    "item": "todo item",
+    "completed": false
+}
+```
+
+## Limitations
+Node.js currently does not support modifying/deleting an existing entity through Table Storage binding. Hence this template is limited to Read and Create operations only.
```

Comment:
As per my comment above, I don't believe the C# binding supports updates either?